docs/source-2.0/aws/aws-iam.rst

```diff
@@ -432,6 +432,11 @@ Each condition key structure supports the following members:
       - ``string``
       - A relative URL path that defines more information about the condition key
         within a set of IAM-related documentation.
+    * - required
+      - ``boolean``
+      - Defines whether a service resolved condition key is required. Not 
+        applicable to request resolved condition keys as the native 
```

Comment:
The "native @required" should be replaced with a `:ref:` to the required trait.

Reply:
```suggestion
        applicable to request resolved condition keys, as the native 
```

Reply:
There should be some sort of validation for if this property is set and the named key is request-resolved. I think  that case should fail and direct users to applying the `@required` trait instead.

Reply:
Still missing the `:ref:` for the required trait.

Reply:
> Still missing the :ref: for the required trait and the validation wasn't added.

Added